doc/lua_api.txt

```diff
@@ -6314,6 +6314,8 @@ object you are working with still exists.
       Default `{x=0, y=0, z=0}`
     * `forced_visible`: Boolean to control whether the attached entity
        should appear in first person. Default `false`.
+    * This command may fail silently (do nothing) when it would result
```

Comment:
this comment is invalid now no ? as you added a warning stream

Reply:
There's no return value or error to indicate the failure, hence it's a silent fail in perspective of the modding API.